README.md

```diff
@@ -28,24 +28,30 @@ You can also find some example notebooks here which should help you with your de
 
 ## Organiser setup
 
-The organisers will effectively execute:
+The organisers will execute (after downloading https://petric.tomography.stfc.ac.uk/data/ to `/path/to/data`):
 
 ```sh
-docker run --rm -it -v data:/mnt/share/petric:ro ghcr.io/synerbi/sirf:edge-gpu
-# or ideally ghcr.io/synerbi/sirf:latest-gpu after the next SIRF release!
-python
+docker run --rm -it -v /path/to/data:/mnt/share/petric:ro -v .:/workdir -w /workdir synerbi/sirf:edge-gpu /bin/bash
+# ... or ideally synerbi/sirf:latest-gpu after the next SIRF release!
+pip install git+https://github.com/TomographicImaging/Hackathon-000-Stochastic-QualityMetrics
```

Comment:
Why not add this to our docker image? But this is fine of course

Reply:
I think it'll change a lot right now, and eventually be baked in to CIL itself, so wouldn't add it to the SIRF image for now.